### ISSUE TITLE
Interview with  ACS

### ISSUE THREAD

**Issue by loretopaulinojr** [loretopaulinojr/Lifestyle#12]
ACS

**Comment by loretopaulinojr** [comment]
Yes, "Jr." would be great to add in the Q&A.
Do you know about how many kilometers the base camp was from Bethel?

Based on the GPS locations, it is around 95 km.
What were the names of the streams in your study site?

Names of the streams? 
The streams were never named formally. I named the sites based on the landscape category.


What kind of information will analyzing the gas samples from the bubble traps tell you?
The rates of ebullition for gases like methane. A rate of which can tell the amount of the analyte that is released into the atmosphere over a period of time.

What research are you doing now and why?
One of my research projects now is looking into a marine bacterial strain database and using genome modeling to investigate metabolites of interest. One group of compounds I am interested in is recalcitrant dissolved organic carbon (RDOC). RDOC are stable organic matter that can persist in the ocean for long periods of time. Little is known about mechanisms related to the production of RDOC by marine bacterial strains so this project will try to fill such knowledge gaps. 

What might your research look like as a professor studying chemical oceanography and why?
As an aspiring chemical oceanographer, I am interested in research that looks into how compounds of interest travel in the ocean. Depending on the analyte, mapping or predicting where the analyte is can become invaluable information to certain communities. But having done some microbial work in my present job, I am leaning towards researching modeling ocean microbiomes in the future, as a possible route. This question is quite hard at the time for my program really opens different possibilities of what I can focus on. But overall, I am interested in the ocean, understanding the ocean again benefits so many communities. As an Asian Pacific Islander, allowing myself to study the ocean is providing a platform for my community to be part of the discussion that we are historically excluded from. To be part of the discussions in the scientific community is to be able to contribute information that can be used to make or influence policies that affect a huge number of people.

She'd also love a few more photos, including:
•	a headshot of you (this could be taken anywhere, it doesn't have to be in Alaska) and 
•	a beaver dam or other signs of beaver activity in Alaska (if you have one).

**Comment by loretopaulinojr** [comment]
### Hometown:
Yigo, Guam (we don’t use the word "hometown" as we call them villages)

### Current position:
Bridget-to-PhD Fellow at Boston University under the Center for Chemical Currencies of a Microbial Planet at Woods Hole Oceanographic Institution

### Education/Degrees:
Bachelor of Science in Chemistry with a Minor in Mathematics

### Previous research projects:
- **Passive Air Sampling for Outdoor and Indoor Volatile and Ionic Per- and Polyfluorinated Alkyl Substances (2022 Rhode Island, URIGSO, mentored by Jitka Becanova and Rainer Lohmann)**
  - Tested and compared four sorbent materials for their ability to accumulate PFAS in air.
  - The performance of each material was evaluated based on the various types of PFAS and the absolute mass of each PFAS accumulated during the deployment period.

- **Investigating Coral Resiliency through Mathematical Modeling (2021 Guam, UOG, mentored by Dr. Oh, Dr. Choi, Dr. Bentlage, and Dr. Aquino)**
  - Created two Mathematical Models:
    - Represented the ability for a coral to recover "Recovery" through a Susceptible, Infected, and Recovered (S.I.R.) model.
    - Represented how resistant the coral is "Resistance" through Game Theory.

### Favorite element or molecule and why:
Hydrogen, I guess it is the way it was introduced to you as the simplest element yet as you go further to senior level chemistry classes. This simple element was used to develop an understanding of things like quantum theory.

### Hobbies:
- Social media
- Running
- Going to the beach

### Most challenging part of your beaver project:
Terrain at the Yukon-Kuskokwim Delta. A normal day would start around 7 am to prepare the equipment needed in the field. The days would usually end around 5 pm, and walking back to camp was quite dreadful.

### Regular Equipment:
- Ropes
- YSI, a portable water sampling meter
- Flags
- GPS
- Bubble Traps
- Syringes, Sample Bottles, Filters, Gas Containers
- Meter sticks
- Field Laptop

### Favorite part of your beaver project:
The sites where the beavers were located. Near the dams, you would see flowers and plants that you don’t really see anywhere else around that terrain.

### Favorite place in Boston:
Boston Public Library

### What you miss most about home:
The beach and its clear warm waters.

### Dream career in 20 years:
Being a professor in chemistry, while doing research in the field of chemical oceanography.

### Big takeaways from Arctic fieldwork experience:
I now believe that I can conduct my own research projects both computationally and in the field.

### Fieldwork vs. Lab work:
I love all the fieldwork that I have been a part of. Compared to lab work, I find fieldwork more engaging.

### Top research priority:
Ocean research

### Challenges faced:
I have faced challenges and barriers as a person geographically as a Pacific Islander.

### Importance of diversity in STEM:
Diversity, inclusion, and equity are important as they foster representation from different backgrounds in crucial discussions.

**Comment by loretopaulinojr** [comment]
If the article gets published:

Email CNAS to help broadcast: 

Hafa adai, 

I am writing this email in how of helping an article published by the American Chemical Society spread to young islanders. To give context, I recently got interviewed by the American Chemical Society about my recent project in the Arctic. Instead of the direction of focusing on my project, I shedded light on some of the challenges that Pacific Islander receive while in the STEM field. 

I am so honored to have been published by ACS as it gives a huge platform to get the message out to the science community. I believe that the problems I talked about in the article should be known and shared to communities who know little of the challenges of being a Pacific Islander in STEM. 

With that said the article gets more encounter based on the metrics. The metrics are based on article views, ale metric and citations. 

Article Views 
are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.

Citations
are the number of other articles citing this article, calculated by Crossref and updated daily.[ Find more information about Crossref citation counts.](https://www.crossref.org/services/cited-by/)

The Altmetric Attention Score
 is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on[ the Altmetric Attention Score and how the score is calculated.](https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated)


I am humbly requesting if there is anyway you may help share the article. This will help not only with the metrics but getting the article out to more communities! 



Message to Polaris:

Hafa Adai 2023 Polaris,

I have recently been interviewed by American Chemical Society (ACS) about my beaver project. In the interview, I shared some of the challenges in the STEM community as a Pacific Islander (PI). These challenges are what I wanted to focus on as ACS, being the platform that it is, can reach many audiences that may not know these obstacles that PIs face in STEM. 

I share this to this program given the goal on diversity. Many organization identify as having diversity, but is there inclusion, is there equity?  Diversity is nothing when these two components are not considered. The article talks about these problems and hope to share you guys the struggles of one of the least represented group in STEM. 

Having spoken to Dr. Rigoberto Hernandez during a presentation, I have been inspired of the work that is trying to close the representation gap in STEM. In his words, he can not do it alone, no one can. With that said, I find myself contributing to this gap through the media. I have no team like Dr. Hernandez, but I use the media to help get the word out. I humbly ask that you all may take a second to read or share the article. 

May this generation give minority groups a positive slope in future data sets. 

Warm regards, 
Loreto Paulino 
Research Fellow at Boston University
8084583880

**Comment by loretopaulinojr** [comment]
The article was published, but now you want it to other communities. 

https://pubs.acs.org/doi/10.1021/acscentsci.4c00192

There is a typo in the paragraph: I tested water upstream and downstream of dams for nitrogen, phosphorus, and carbon. While I found no significant differences, further research with larger sample sizes might show differences. I also took gas samples with bubble traps─a syringe attached to a funnel on the water surface─to measure nitrogen, carbon dioxide, and methane releases into the atmosphere, and I’m still waiting to analyze them.


Message Robin, that I did not take gas samples to measure nitroge. NITROGEN IS INTERT.
